AKostiv_Flask_App/README.md

```diff
@@ -0,0 +1 @@
+# GCI2017-Flask-App
```

Comment:
@AKostiv8 You have done a great job 🥇 
Please add content in the readme, Tell about the project, paste some screenshot, add working deployment link, add how to run this project if a new commer wants to run your project.
Thanks